.gitlab-ci.yml

```diff
@@ -84,6 +84,7 @@ download-single-step-artifacts:
   stage: package
   image: registry.ddbuild.io/docker:20.10.13-gbi-focal
   tags: [ "arch:amd64" ]
+  needs: []
```

Comment:
This change is already in master. The download job doesn't depend on anything else